### ISSUE TITLE
Django Admin & Autocomplete Endpoints

### ISSUE THREAD

**Issue by mbhassan** [silverapp/silver#667]
Issue in autocomplete endpoints when using django admin.

```
File "/usr/local/lib/python3.7/site-packages/silver/views.py", line 180, in get_queryset
django_1        |     if not (self.request.user.is_authenticated() and self.request.user.is_staff):
django_1        | TypeError: 'bool' object is not callable
```

self.request.user.is_authenticated() is not a function but instead an attribute. 

Resolution:
Should be changed to self.request.user.is_authenticated

**Comment by bogdanpetrea** [comment]
Hey @mbhassan , thanks for the heads-up!

You are right, from the Django docs:
> Changed in Django 1.10:
In older versions, this was a method. Backwards-compatibility support for using it as a method will be removed in Django 2.0.

Would you be willing to open a PR with a fix?

**Comment by mbhassan** [comment]
sure. ill open a pr after fixing the bug.

**Comment by mbhassan** [comment]
@bogdanpetrea i'm going to add some tests into the admin folder for the autocomplete endpoints.

**Comment by bogdanpetrea** [comment]
Closed in #668.
